src/modules/lua_mtev.c

```diff
@@ -2776,6 +2777,20 @@ nl_print(lua_State *L) {
   lua_call(L, n, 0);
   return 0;
 }
+/*! \lua len = mtev.printto(facility)
+\param facility is an mtev log stream
+
+This function sets the mtev.print output facility.
+*/
+static int
+nl_printto(lua_State *L) {
+  int n = lua_gettop(L);
+  if(n != 1) luaL_error(L, "mtev.printto(facility)");
+  char *tofree = print_tgt;
+  print_tgt = strdup(luaL_checkstring(L,1));
+  free(tofree);
+  return 0;
+}
```

Comment:
I wonder about the use-case of this. I have personally never used mtev.print(). Instead:
- print() -- from lua; or
- mtev.log()

Is this change in order to patch existing modules/apps?

Reply:
libmtev's lua support replaces global `print` with `mtev.print` which defaults to "error".

This function just lets one (at an application level) change that default...

Since luamtev is run from the command line, it makes sense to use stdout.  When lua is running in a regular metv app, stdout makes little since.  But perhaps `error/lua/print` might make more sense as a default?  Something to think about later.

Reply:
that makes sense.